pb/satellite.proto

```diff
@@ -0,0 +1,22 @@
+// Copyright (C) 2021 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+syntax = "proto3";
+option go_package = "storj.io/common/pb";
+
+package satellite;
+
+import "node.proto";
+
+// Config contains public satellite configuration that may be useful to storage nodes or uplinks.
+service Config {
+    // MinimumStorageNode returns minimum versions required for storage nodes.
+    rpc MinimumStorageNode(MinimumStorageNodeRequest) returns (MinimumStorageNodeResponse);
+}
+
+message MinimumStorageNodeRequest {}
+
+message MinimumStorageNodeResponse {
+    // upload is the minimum storage node version that are considered for uploading data.
+    node.NodeVersion upload = 1;
```

Comment:
Per the PR description 

> all minimum selection versions from trusted satellites will be compared and newest version will be shown
as version on which node still won't be disqualified, but would not receive new data, just storing."

The name "upload" seems the opposite of this, we usually call this download. To avoid this ambiguity maybe call it "storeNewPieces"?